src/libraries/System.Net.Http/tests/StressTests/HttpStress/LogHttpEventListener.cs

```diff
@@ -116,10 +118,8 @@ protected override async void OnEventWritten(EventWrittenEventArgs eventData)
                 sb.Append(eventData.PayloadNames?[i]).Append(": ").Append(eventData.Payload[i]);
             }
             sb.Append(Environment.NewLine);
-            string s = sb.ToString();
-            sb.Clear();
-            Interlocked.Exchange(ref _cachedStringBuilder, sb);
-            await _messagesChannel.Writer.WriteAsync(s, _stopProcessing.Token);
+            await _messagesChannel.Writer.WriteAsync(sb.ToString(), _stopProcessing.Token);
+            _stringBuilderPool.Return(sb);
```

Comment:
Does `StringBuilderPooledObjectPolicy` clear the `StringBuilder` now?

Reply:
Yep, taken from this blog post: https://docs.microsoft.com/en-us/aspnet/core/performance/objectpool?view=aspnetcore-5.0
And confirmed by this:
https://source.dot.net/#Microsoft.Extensions.ObjectPool/StringBuilderPooledObjectPolicy.cs,30f83640ea3a7033,references